@ecomplus/storefront-components/src/js/TheProduct.js

```diff
@@ -283,6 +285,10 @@ export default {
 
     isKit () {
       return this.body.kit_composition && this.body.kit_composition.length
+    },
+
+    kitHasVariations () {
```

Comment:
Booleano é mais claro com **prefixo** `is`, `has`, `was`...
Tipo `isKitWithVariations`